data/faq/application_template.md

```diff
@@ -0,0 +1,86 @@
+coala 2017 Application Template
+===============================
+
+Student Info
+------------
+
+- Name:
+- GitHub username:
+- Alternative-/Nickname:
+- Email:
+- Telephone:
+- Time Zone:
+- GSoC blog RSS feed URL:
+    You will have to write a blog post every week during GSoC.
+
+Code Sample
+-----------
+
+Link all your contributions to coala and any other open source project here.
+For example, to list your contributions to coala you can mention:
+ - https://github.com/coala/coala/commits/master?author=YOUR_USER_NAME for coala repo
+ - https://github.com/coala/coala-bears/commits/master?author=YOUR_USER_NAME for coala-bears repo
```

Comment:
also there are other repos, you should mention those should be listed too otherwise people will ask